client/src/mvc/tool/tool-form.js

```diff
@@ -49,6 +50,20 @@ const View = Backbone.View.extend({
                                     window.location = getAppRoot();
                                     return;
                                 }
+
+                                const user_version = options.id.replace(/.*\//, "");
+                                if (!data.versions.includes(user_version)) {
+                                    Toast.error(
```

Comment:
```suggestion
                                    Toast.warning(
```

Reply:
Since there was actually an alternative version, let's not throw the error Toast, but a warning?

Reply:
sure